app/styles/main.scss

```diff
@@ -4,3 +4,7 @@
 
 @import 'containers/containers';
 @import 'components/components';
+
+body{
+  background:yellow;
```

Comment:
It would be better if we don't have variables with name "yellow". Instead it should be "backgroundcolor" or something descriptive. The variable has a name, yellow is the value.